eox_hooks/tests/test_receivers.py

```diff
@@ -0,0 +1,57 @@
+"""This file contains all the test for the receivers.py file.
+
+Classes:
+    SingalsTestCase: Test Receivers.
+"""
+from django.test import TestCase, override_settings
+from mock import Mock, patch
+
+from eox_hooks.receivers import hooks_handler
+
+
+@override_settings(
+    EOX_HOOKS_DEFINITIONS={
```

Comment:
Why is this here?, if there is a file for this kind a things

Reply:
changed